be/src/olap/tablet_meta.cpp

```diff
@@ -935,6 +935,7 @@ void TabletMeta::delete_stale_rs_meta_by_version(const Version& version) {
                 // remove rowset delete bitmap
                 delete_bitmap().remove({(*it)->rowset_id(), 0, 0},
                                        {(*it)->rowset_id(), UINT32_MAX, 0});
+                delete_bitmap().remove_rowset_cache_version((*it)->rowset_id());
```

Comment:
Add a DCHECK at the end of the fucntion, to check that the size of _rowset_cache_version is smaller than the size of `_rs_metas.size() + _stale_rs_metas.size()` to avoid mem leak